db/migrate/20231101014117_create_parties.rb

```diff
@@ -2,13 +2,13 @@ class CreateParties < ActiveRecord::Migration[7.0]
   def change
     create_table :parties do |t|
       t.string :access_code
-      t.integer :max_rating, default: nil
-      t.integer :max_duration, default: nil
+      t.integer :max_rating, default: -1
+      t.integer :max_duration, default: -1
```

Comment:
This totally make sense that we need to utilize an integer over `nil` since we have the data type as an integer.